node/tests/ServerModules.test.ts

```diff
@@ -717,228 +691,306 @@ describe("Server Module Tests", () => {
                 expect(
                     await GlideJson.resp(client, "nonexistent_key"),
                 ).toBeNull();
-            },
-        );
-    });
-
-    describe("GlideFt", () => {
-        let client: GlideClusterClient;
-
-        afterEach(async () => {
-            await flushAndCloseClient(true, cluster.getAddresses(), client);
-        });
-
-        it("ServerModules check Vector Search module is loaded", async () => {
-            client = await GlideClusterClient.createClient(
-                getClientConfigurationOption(
-                    cluster.getAddresses(),
-                    ProtocolVersion.RESP3,
-                ),
-            );
-            const info = await client.info({
-                sections: [InfoOptions.Modules],
-                route: "randomNode",
             });
-            expect(info).toContain("# search_index_stats");
-        });
-
-        it("Ft.Create test", async () => {
-            client = await GlideClusterClient.createClient(
-                getClientConfigurationOption(
-                    cluster.getAddresses(),
-                    ProtocolVersion.RESP3,
-                ),
-            );
-
-            // Create a few simple indices:
-            const vectorField_1: VectorField = {
-                type: "VECTOR",
-                name: "vec",
-                alias: "VEC",
-                attributes: {
-                    algorithm: "HNSW",
-                    type: "FLOAT32",
-                    dimensions: 2,
-                    distanceMetric: "L2",
-                },
-            };
-            expect(
-                await GlideFt.create(client, uuidv4(), [vectorField_1]),
-            ).toEqual("OK");
-
-            expect(
-                await GlideFt.create(
-                    client,
-                    "json_idx1",
-                    [
+        },
+    );
+
+    describe.each([ProtocolVersion.RESP2, ProtocolVersion.RESP3])(
+        "GlideFt",
+        (protocol) => {
+            let client: GlideClusterClient;
+
+            afterEach(async () => {
+                await flushAndCloseClient(true, cluster.getAddresses(), client);
+            });
+
+            it("ServerModules check Vector Search module is loaded", async () => {
+                client = await GlideClusterClient.createClient(
+                    getClientConfigurationOption(
+                        cluster.getAddresses(),
+                        protocol,
```

Comment:
please revert for VSS
we had trouble with tests running too many times and creating too many indices. there is no value to running FT.CREATE against RESP2 and RESP3.  FT.INFO can be run against both RESP2 and RESP3, but not CREATE

Reply:
Updated